scripts/demoScripts/reportsMockScript.js

```diff
@@ -0,0 +1,175 @@
+const Client = require('node-rest-client').Client;
```

Comment:
FFR: You probably wanted to `git mv scripts/reportsMockScript.js  scripts/demoScripts/reportsMockScript.js`. This would have kept the history of the file (delete / create doesn't preserve file history).